DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata.py

```diff
@@ -248,7 +278,8 @@ def removeMetadata(self, dpath, metadata, credDict):
 
   def setMetaParameter(self, dpath, metaName, metaValue, credDict):
     """ Set an meta parameter - metadata which is not used in the the data
-        search operations
+        search operations.
+        Modified by JM: make metaName Dirac group aware.
```

Comment:
Comments like that are suitable for release notes but for the in-code documentation

Reply:
OK